src/pages/Submission/index.js

```diff
@@ -1,26 +1,43 @@
 import React, { useEffect, useState } from 'react'
-import { getLivesiteDoc } from '../../utility/firebase'
+import { getLivesiteDoc, applicantsRef } from '../../utility/firebase'
 import Submission from '../../components/Submission'
 
+// TODO: Get from firebase auth
+const USER_ID = 'aIwA36q0kOw7rDDlCkB2'
+
 export default () => {
   const [isSubmissionsOpen, setIsSubmissionsOpen] = useState(false)
+  const [project, setProject] = useState()
+  const [feedback, setFeedback] = useState([])
 
-  // TODO: use firebase data
-  // eslint-disable-next-line no-unused-vars
-  const [project, setProject] = useState({
-    description: 'This project is a project that is very cool!',
-    imgUrl: 'https://img.youtube.com/vi/PQgHXPGoKwg/maxresdefault.jpg',
-    devpost: 'https://devpost.com/software/impostor',
-    title: 'Imposter',
-    teamName: 'H4ckHouse',
-  })
-
-  // TODO: use firebase data
-  // eslint-disable-next-line no-unused-vars
-  const [feedback, setFeedback] = useState([
-    'Very good!',
-    'Wow this project changed my life like how is this real omg i would give my life for this.',
-  ])
+  useEffect(() => {
+    ;(async () => {
```

Comment:
what does the semicolon here do? comments might be nice here

Reply:
I am not sure, precommit hook added it 🤣 🤔

Reply:
https://stackoverflow.com/questions/17978883/what-is-the-purpose-of-a-semicolon-before-an-iife